gufe/protocols/protocoldag.py

```diff
@@ -17,6 +17,13 @@
     ProtocolUnit, ProtocolUnitResult, ProtocolUnitFailure, Context
 )
 
+from ..storage.storagemanager import StorageManager
+from ..storage.externalresource.filestorage import FileStorage
+from ..storage.externalresource.base import ExternalStorage
```

Comment:
can we fix the namespace of `storage` so these are all available at the same level?  `from ..storage import StorageManager, FileStorage, etc`